package.json

```diff
@@ -57,7 +57,8 @@
     "react-native-windows": "^0.72.19",
     "react-test-renderer": "^18.2.0",
     "semantic-release": "^22.0.7",
-    "typescript": "^5.2.2"
+    "typescript": "^5.2.2",
+    "react-native-fs": "^2.20.0"
```

Comment:
**react-native-fs** I believe it's best to avoid dependencies on other projects and let the end user decide which package to use.

Reply:
Ok. Thanks. It's used only to demo the implementation.